modules/installation-special-config-custom-image.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-custom-image_{context}"]
+= Generating and applying a custom override base image
+
+In {product-title} {product-version}, the {op-system-first} component
+is now shipped as a standard container image, and it is supported to
+create customized versions using standard container build tooling.
+
+To find the container image associated with a specific {product-title}
+version, use `oc adm release info --image-for rhel-coreos-8 <image>`
+on the release image reported by `oc get clusterversion`.
+
+== Generating a container image with overrides
+
+This example `Dockerfile` demonstrates using `rpm-ostree override replace`
+to generate a container image that replaces the `NetworkManager` package:
+
+```
+FROM <operating system base>
+RUN rpm-ostree override replace https://example.com/rhel-8-mirror//BaseOS/x86_64/os/Packages/NetworkManager-1.40.0-1.el8.x86_64.rpm \
+    rpm-ostree cleanup -m && \
+    ostree container commit
+````
+
+Push this to a registry, again using any tools that operate on container images.
+
+== Applying the custom container image
+
+
+. Create a MachineConfig file which overrides `osImageURL`, referencing
+your custom image.
+
++
+----
+$ cat << EOF > ./99-masters-chrony-configuration.yaml
```

Comment:
chrony is missleading here if the example talks about replacing NM

Reply:
not only chrony, but `masters` too.